src/main/java/io/kestra/plugin/weaviate/BatchCreate.java

```diff
@@ -0,0 +1,97 @@
+package io.kestra.plugin.weaviate;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.weaviate.client.WeaviateClient;
+import io.weaviate.client.base.Result;
+import io.weaviate.client.base.WeaviateErrorMessage;
+import io.weaviate.client.v1.batch.model.ObjectGetResponse;
+import io.weaviate.client.v1.data.model.WeaviateObject;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@SuperBuilder
+@ToString
+@Getter
+@EqualsAndHashCode
+@NoArgsConstructor
+@Schema(
+    title = "Batch create request to Weaviate database.",
+    description = "Create batch request to Weaviate database. If schema does not exist, schema will automatically be created"
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Send batch object creation request to a Weaviate database",
+            code = {
+                "host: localhost:8080",
+                "apiKey: some_api_key",
+                "className: WeaviateObject"
+            }
+        )
+    }
+)
+public class BatchCreate extends WeaviateConnection implements RunnableTask<BatchCreate.Output>, BatchCreateInterface {
+
+    protected String className;
+
+    @Builder.Default
+    protected List<Map<String, Object>> parameters = List.of(new HashMap<>());
+
+    @Override
+    public BatchCreate.Output run(RunContext runContext) throws Exception {
+        WeaviateClient client = connect(runContext);
+
+        List<WeaviateObject> objects = new ArrayList<>();
+
+        for (Map<String, Object> properties : parameters) {
+            objects.add(WeaviateObject.builder().className(className).properties(properties).build());
+        }
+
+        Result<ObjectGetResponse[]> result = client.batch().objectsBatcher().withObjects(objects.toArray(WeaviateObject[]::new)).run();
+
+        if (result.hasErrors()) {
+            String message = result.getError().getMessages().stream()
+                .map(WeaviateErrorMessage::getMessage)
+                .collect(Collectors.joining(", "));
+
+            throw new IOException(message);
+        }
+
+        List<ObjectGetResponse> responses = List.of(result.getResult());
+
+        return Output.builder()
+            .className(responses.stream().map(ObjectGetResponse::getClassName).toList())
+            .properties(responses.stream().map(ObjectGetResponse::getProperties).toList())
```

Comment:
As said below, created objects should be stored in the internal storage and the URI of the file containing the created data returned (see [this](https://github.com/kestra-io/plugin-aws/commit/c2b188c3c2fdaff2b56630d2d22176c7fdc517ec#diff-668ca875546bf9c9f99bf1bb34686d4c7fa1517151dbd1cc2e9da82647e085fdR180-R195))